26-register_publication/1625-creator_navigates_to_the_reference_tab_and_selects_reference_subtype_corrigendum.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Creator navigates to the Reference tab and selects Reference subtype "Corrigendum"
+
+    @1625
+    Scenario: Creator navigates to the Reference tab and selects Reference subtype "Corrigendum"
+        Given that a Creator navigates to the Reference tab
+        And they have selected "Contribution to journal" as Reference type
+        When they select "Corrigendum" as Reference Subtype
+        Then they see mandatory fields:
+            | Link to Journal article |
+        And they see optional fields:
+            | DOI            |
+            | Volume         |
+            | Issue          |
+            | Pages from     |
+            | Pages to       |
+            | Article number |
+        And they see a disabled field for Journal based on selected Journal article
```

Comment:
Skal vi skrive disse scenariene på samme form? slik at vi har samme mal? Ref #53 :
```
        Given Creator begins registering a Publication in the Wizard
        When they navigate to the Reference tab
        And they select the Reference type "Contribution to journal"
```

Reply:
jeg er ikke så veldig glad i `optional fields`. Er det optional som i at de kan/kan ikke finnes, eller optional som i at de ikke _må_ fylle de ut? Synes ikke egentlig at det er nødvendig med ordet `optional` og `mandatory`. Føler det er mer valideringsregler

Reply:
optional som i validering ja, men DOI vises heller ikke om den ikke har det da...
jeg forventer egentlig at jeg skal kunne gjøre ferdig hele Corrigengum bare ved å se denne storyen. om vi har en story som lister opp alle felter, og en annen story som dekker validering virker det mer som om vi skriver stories på sub-task nivå, og at vi må fullføre flere stories for å bli ferdig med hver enkelt type. jeg tenker at validering bør være en subtask i denne storyen og ikke i en annen story. om vi skal endre felter i fremtiden må vi evt oppdatere flere stories også

Reply:
ja, jeg ser den. men føler samtidig at vi bør begrense antall features i et scenario. burde egentlig ha en tredjeperson til å se på dette siden vi er uenig. @Eiriknil ?

Reply:
Jeg ser at det er et problem at vi har splittet opp storyene i en (cucumber-) Feature pr scenario. Tror det hadde vært lettere å få oversikt hvis en story var en cucumber-Feature med flere scenarioer.

Reply:
Min løsning vil være å droppe `optional` og legge til et `Scenario` i samme `Feature` som tar seg av validering.

Reply:
Ja, godt poeng. da burde en story i Jira tilsvare en Feature her da, så kan vi ha subtasks i samme Jira-story på tvers av Scenarios?